lib/spack/docs/example_files/test.yaml

```diff
@@ -0,0 +1,22 @@
+---
+spec-set:
+    include: [ ape, atompaw, transset]
+    exclude: [binutils,tk]
```

Comment:
I forget if this is the case, but if there is an include list *and* an exclude list, the thing should start with what's in `include` and exclude should exclude from *that* set.  So in this example, the exclude list wouldn't be necessary.

Reply:
Makes sense, I'll keep that in mind.  In the sample `release` file I provided (just as a start), I just left them both empty for the moment.  I figured I would start populating the `exclude` list as I discover that some things don't work together.